doc/whats_new.rst

```diff
@@ -38,6 +40,12 @@ API
 
     - Make the goodness of fit (GOF) of the dipoles returned by :func:`mne.beamformer.rap_music` consistent with the GOF of dipoles returned by :func:`mne.fit_dipole` by `Alex Gramfort`_.
 
+    - :class:`mne.decoding.SlidingEstimator` will now replace `mne.decoding.TimeDecoding` to make it generic and fully compatible with scikit-learn, by `Jean-Remi King`_ and `Alexandre Gramfort`_
```

Comment:
double-backticks on the second one to render as code instead of broken Sphinx link